content/open-source/apko/getting-started-with-apko.md

```diff
@@ -59,11 +59,11 @@ You should get output similar to this:
  /_/   \_\ |_|     |_|\_\  \___/
 apko
 
-GitVersion:    v0.5.0
-GitCommit:     71cdb2460aece396d00dd4ea47d7d7b5eab3a37d
-GitTreeState:  clean
-BuildDate:     2022-07-18T16:59:54
-GoVersion:     go1.18.3
+GitVersion:    v0.6.0
+GitCommit:     unknown
```

Comment:
Do we want all these "unknown"s in here?

Reply:
yeah I don't know why, it used to show the correct info but this latest version apparently doesn't. I will try to cut the output a bit to suppress these since it's probably a temporary glitch on the image building side (this is running apko via docker)